src/core/createEventManager.js

```diff
@@ -116,6 +116,9 @@ export default ({
         })
         .then(() => {
           payload.addEvent(event);
+          return consent.awaitConsent();
```

Comment:
This is a tricky issue. The problem with this change is that it could lead to multiple ECIDs. Consider this scenario:

1. New user visits page configured with defaultConsent = "pending"
2. Server side call occurs. Konductor generates ECID1.
3. applyHandles is called, but is not processed because consent is pending.
4. setConsent is called. Konductor generates ECID2.
5. Web SDK now processes handles from applyHandles call.

Reply:
I think you should go back to not waiting for consent to process the handles.